src/Css.elm

```diff
@@ -3375,3 +3386,50 @@ inset =
 outset : Value { provides | outset : Supported }
 outset =
     Value "outset"
+
+-- TEXT TRANSFORM --
+
+{-| Sets [`text-transform`](https://css-tricks.com/almanac/properties/t/text-transform/).
+
+    textTransform capitalize
+    textTransform uppercase
+
+-}
+textTransform :
+    Value
+        { capitalize : Supported
+        , uppercase : Supported
+        , lowercase : Supported
+        , full-width : Supported
```

Comment:
Close! This should be `fullWidth : Supported` to match the name on the `elm-css` side.

The idea is that if someone gets an error message because they passed the wrong type to `textTransform`, the error message should list all the values that are permitted - such as `fullWidth`!